### PR TITLE
Deal with dynamic admins

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

File: repondeur/zam_repondeur/scripts/admin.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+"""
+Add an allowed pattern to the email authentication whitelist
```
Comment (ronnix) [inline repondeur/zam_repondeur/scripts/admin.py]: Oops

File: admin/tasks/deploy.py [hunk 2]
```diff
@@ -292,6 +289,25 @@ def whitelist_check(ctx, email):
...
 
+@task
+def admin_list(ctx):
+    cmd = f"{venv_dir}/bin/zam_admin production.ini#repondeur list"
+    res = ctx.sudo(f'bash -c "cd {app_dir} && {cmd}"', user=user)
+    return res.stdout
+
+
+@task
+def admin_set(ctx, email):
```
Comment (ronnix) [inline admin/tasks/deploy.py]: Maybe grant / revoke instead of set / unset ?

File: repondeur/zam_repondeur/resources.py [hunk 3]
```diff
@@ -75,19 +75,30 @@ class Root(Resource):
...
         self.add_child(WhitelistCollection(name="whitelist", parent=self))
+        self.add_child(AdminsCollection(name="admins", parent=self))
         self.add_child(DossierCollection(name="dossiers", parent=self))
 
 
 class WhitelistCollection(Resource):
     __acl__ = [(Allow, "group:admins", "manage"), (Deny, Everyone, "manage")]
 
-    def models(self, *options: Any) -> List[Dossier]:
+    def models(self, *options: Any) -> List[AllowedEmailPattern]:
```
Comment (ronnix) [inline repondeur/zam_repondeur/resources.py]: 👍

File: repondeur/zam_repondeur/scripts/admin.py (hunk 1 above)
Comment (davidbgk) [reply repondeur/zam_repondeur/scripts/admin.py]: ¡Caramba! Je suis découvert :)

File: admin/tasks/deploy.py (hunk 2 above)
Comment (davidbgk) [reply admin/tasks/deploy.py]: Good idea, I'll do that tomorrow.

**Pull request merged**